tst/TurnConnectionFunctionalityTest.cpp

```diff
@@ -127,8 +127,8 @@ namespace com { namespace amazonaws { namespace kinesis { namespace video { name
         turnPeerAddr.port = (UINT16) getInt16(8080);
         turnPeerAddr.family = KVS_IP_FAMILY_TYPE_IPV4;
         turnPeerAddr.isPointToPoint = FALSE;
-        // random peer 10.1.1.1, we are not actually sending anything to it.
-        turnPeerAddr.address[0] = 0x0A;
+        /* random peer 77.1.1.1, we are not actually sending anything to it. */
```

Comment:
Wonder if we can #define these and use these directly?

Reply:
I dont think we can without making it excessively hard to read.

Reply:
I agree.